.travis.yml

```diff
@@ -10,13 +10,14 @@ jobs:
       name: "Docker build Test"
       script:
         - sudo rm /usr/local/bin/docker-compose
-        - curl -L https://github.com/docker/compose/releases/download/1.22.0/docker-compose-Linux-x86_64 > docker-compose
+        - curl -L https://github.com/docker/compose/releases/download/1.25.0/docker-compose-Linux-x86_64 > docker-compose
         - chmod +x docker-compose
         - sudo mv docker-compose /usr/local/bin
         - cd docker
         - docker build --rm -t imlteam/manager-service-base:5.1 -f base.dockerfile ../
-        - docker build --rm -t imlteam/manager-nginx -f nginx.dockerfile ../
-        - docker-compose build
+        - docker build --rm -t imlteam/manager-nginx:5.1 -f nginx.dockerfile ../
+        - docker build --rm -t rust-iml-base -f rust-base.dockerfile ../
```

Comment:
Should this be `imlteam/rust-iml-base`?

Reply:
I purposely left that off because it's not published. I don't think we should be publishing this (yet).

Reply:
ok